docs/release_notes.md

```diff
@@ -6,20 +6,24 @@
 
 - Attachment to geometry constraints in AdnMuscle and AdnRibbonMuscle solvers. Multiple target geometries are supported.
 - Slide on geometry constraints in AdnMuscle and AdnRibbonMuscle solvers. Multiple target geometries are supported.
+- Shape preservation constraints integrated in all solvers.
```

Comment:
I would specify here that Shape Preservation constraints have been created for this release and integrated them in all solvers. Like it is it sounds like we already had them in some solvers and just have integrated them in all of them.

Reply:
I would list the solver names in this case

Reply:
Specified the solver names.

Reply:
Please correct the typo, that "in" is repeated twice.

Reply:
Fixed.